src/tracing/mod.rs

```diff
@@ -129,6 +129,12 @@ impl<P: EthereumProvider + Send + Sync + Clone> Tracer<P> {
         let mut db = self.db;
 
         while let Some(tx) = transactions.next() {
+            // Check if the transaction has `isRunOutOfResources` set to true in the `other` field.
+            if tx.other.get("isRunOutOfResources") == Some(&serde_json::Value::Bool(true)) {
+                // Skip this transaction if `isRunOutOfResources` is true.
```

Comment:
I think we need to return a default tracing result here, otherwise we will have issues